### ISSUE TITLE
wordinator crashes with `ExceptionInInitializerError`

### ISSUE THREAD

**Issue by larsga** [drmacro/wordinator#98]
On some inputs wordinator crashes as follows:

```
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
+ 2023-02-05 15:45:21,289 [INFO ] Input document or directory='table-stuff-20230114-Holman.swpx'
+ 2023-02-05 15:45:21,290 [INFO ] Output directory           ='output.docx'
+ 2023-02-05 15:45:21,290 [INFO ] DOTX template              ='CENELEC-simpletemplate.dotx'
+ 2023-02-05 15:45:21,290 [INFO ] XSLT template              =Not specified
+ 2023-02-05 15:45:21,290 [INFO ] Catalog                    =Not specified
+ 2023-02-05 15:45:21,290 [INFO ] Chunk level                ='root'
+ 2023-02-05 15:45:21,599 [INFO ] Generating DOCX file "/Users/larsga/Desktop/table-stuff-20230114-Holman/output.docx"
+ 2023-02-05 15:45:21,601 [ERROR] Unexpected ExceptionInInitializerError: null
java.lang.ExceptionInInitializerError
	at org.wordinator.xml2docx.MakeDocx.handleSingleSwpxDoc(MakeDocx.java:325)
	at org.wordinator.xml2docx.MakeDocx.handleCommandLine(MakeDocx.java:199)
	at org.wordinator.xml2docx.MakeDocx.main(MakeDocx.java:86)
Caused by: java.lang.UnsupportedOperationException: No class provided, and an appropriate one cannot be found.
	at org.apache.logging.log4j.LogManager.callerClass(LogManager.java:573)
	at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:598)
	at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:585)
	at org.wordinator.xml2docx.generator.DocxGenerator.<clinit>(DocxGenerator.java:378)
	... 3 more
```

The cause is not exactly clear, but apparently to do with log4j trying to get the caller class. Which it shouldn't really have to do, anyway.

**Comment by larsga** [comment]
Merging that PR will solve this issue.

**Comment by larsga** [comment]
Fixed by merging #99
